src/utils/registry.ts

```diff
@@ -562,6 +562,25 @@ export class Registry {
         else
           throw new Error(`ERROR: Playwright does not support installing ${executable.name}`);
       }
+    } catch (e) {
+      if (e.code === 'ELOCKED') {
+        const rmCommand = process.platform === 'win32' ? 'rm -R' : 'rm -rf';
+        throw new Error('\n' + wrapInASCIIBox([
+          `An active lockfile is found at:`,
+          ``,
+          `  ${lockfilePath}`,
+          ``,
+          `Either:`,
+          `- wait a few minutes if other Playwright is installing browsers in parallel`,
+          `- remove lock manually with:`,
+          ``,
+          `    ${rmCommand} ${lockfilePath}`,
+          ``,
+          `<3 Playwright Team`,
+        ].join('\n'), 1));
+      } else {
+        throw e;
+      }
```

Comment:
```suggestion
      }
      throw e;
```
nit

Reply:
has to be in parenthesis since the first `if` clause is multi-line.

Reply:
ah, i'd rather have it this way i guess!